packages/falso/src/tests/clothing-size.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { randClothingSize } from '../lib/clothing-size';
+
+describe('clothingSize', () => {
+  it('should create', () => {
+    expect(randClothingSize).toBeTruthy();
+  });
+  it('should create one clothing size', () => {
+    expect(randClothingSize()).toBeTruthy();
```

Comment:
Please test the return value of the function (make sure it's a string) instead of a general truth 🙂